src/app/pages/HomePage/HomePage.tsx

```diff
@@ -11,24 +11,34 @@ import {
 import Curation from '../../components/Curation';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import styles from './index.styles';
+import MetadataContainer from '../../components/Metadata';
 
 interface HomePageProps {
   pageData: {
     pageType: string;
     id?: string;
     title: string;
     curations: CurationData[];
+    description?: string;
```

Comment:
On second thoughts, we might just need to set this to be a mandatory instead of optional prop, and therefore don't need to default it when destructuring